### PR TITLE
Update si.m3u

### LINKED ISSUE #7793: Add: Boomerang HD | (Slovenia)
### Channel Name

Boomerang

### Channel ID

_No response_

### Website

streamstat.net/videoplayer.cgi?sid=471583162&ext=.m3u8

### Stream URL

http://194.163.179.246/slovenci/djeciji/boomerang.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #7794: Add: HBO HD | (Slovenia)
### Channel Name

HBO

### Channel ID

_No response_

### Website

_No response_

### Stream URL

http://194.163.179.246/slovenci/filmski/hbohd.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #7795: Add: HBO 2 HD | (Slovenia)
### Channel Name

HBO 2

### Channel ID

_No response_

### Website

_No response_

### Stream URL

http://194.163.179.246/slovenci/filmski/hbo2hd.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #7796: Add: HBO 3 HD | (Slovenia)
### Channel Name

HBO 3

### Channel ID

_No response_

### Website

_No response_

### Stream URL

http://194.163.179.246/slovenci/filmski/hbo3hd.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### LINKED ISSUE #7801: Add: Cartoon Network | (Slovenia)
### Channel Name

Cartoon Network 

### Channel ID

_No response_

### Website

_No response_

### Stream URL

http://194.163.179.246/slovenci/djeciji/cartoon.m3u8

### Notes

_No response_

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

### PR DESCRIPTION
**Pull request by freearhey**
Added new links from issues.

Closes #7801, closes #7796, closes #7795, closes #7794, closes #7793

### CODE REVIEW

File: streams/si.m3u [hunk 1]
```diff
@@ -19,3 +19,13 @@ https://16-rtvslo-tv-slo3-int.cdn.eurovisioncdn.net/playlist.m3u8
 https://stream.veseljak.tv/hls/stream.m3u8
 #EXTINF:-1 tvg-id="" status="",History Channel (1080p)
 https://streamer12.xploretv.si/qdcnm4/__cl/cg:prod/__c/A1_SI_HISTORYHD_ott/__op/dash-default/__dci/__f/index.m3u8?redirect=true&admin=xploreTv_test_user
+#EXTINF:-1 tvg-id="",Cartoon Network
+http://194.163.179.246/slovenci/djeciji/cartoon.m3u8
```
Comment (LaneSh4d0w) [inline streams/si.m3u:23]: https://streamer12.xploretv.si/qdcnm4/__cl/cg:prod/__c/A1_SI_CARTOON_ott/__op/dash-default/__dci/__f/index.m3u8?redirect=true&admin=xploreTv_test_user

Comment (LaneSh4d0w) [inline streams/si.m3u:23]:
```suggestion
https://streamer12.xploretv.si/qdcnm4/__cl/cg:prod/__c/A1_SI_CARTOON_ott/__op/dash-default/__dci/__f/index.m3u8?redirect=true&admin=xploreTv_test_user
```

File: streams/si.m3u [hunk 2]
```diff
@@ -19,3 +19,13 @@ https://16-rtvslo-tv-slo3-int.cdn.eurovisioncdn.net/playlist.m3u8
...
+#EXTINF:-1 tvg-id="",HBO 3 HD
+http://194.163.179.246/slovenci/filmski/hbo3hd.m3u8
+#EXTINF:-1 tvg-id="",HBO 2 HD
+http://194.163.179.246/slovenci/filmski/hbo2hd.m3u8
```
Comment (LaneSh4d0w) [inline streams/si.m3u:27]:
```suggestion
https://streamer12.xploretv.si/qdcnm4/__cl/cg:prod/__c/A1_SI_HBOCOMEDYHD_ott/__op/dash-default/__dci/__f/index.m3u8?redirect=true&admin=xploreTv_test_user
```

File: streams/si.m3u [hunk 3]
```diff
@@ -19,3 +19,13 @@ https://16-rtvslo-tv-slo3-int.cdn.eurovisioncdn.net/playlist.m3u8
...
+#EXTINF:-1 tvg-id="",HBO HD
+http://194.163.179.246/slovenci/filmski/hbohd.m3u8
```
Comment (LaneSh4d0w) [inline streams/si.m3u:29]:
```suggestion
https://streamer12.xploretv.si/qdcnm4/__cl/cg:prod/__c/A1_SI_HBOHD_ott/__op/dash-default/__dci/__f/index.m3u8?redirect=true&admin=xploreTv_test_user
```

File: streams/si.m3u [hunk 4]
```diff
@@ -19,3 +19,13 @@ https://16-rtvslo-tv-slo3-int.cdn.eurovisioncdn.net/playlist.m3u8
...
+#EXTINF:-1 tvg-id="",Boomerang HD
+http://194.163.179.246/slovenci/djeciji/boomerang.m3u8
```
Comment (LaneSh4d0w) [inline streams/si.m3u:31]:
```suggestion
https://streamer12.xploretv.si/qdcnm4/__cl/cg:prod/__c/A1_SI_BOOMERANG_ott/__op/dash-default/__dci/__f/index.m3u8?redirect=true&admin=xploreTv_test_user
```

**Review by BellezaEmporium — APPROVED** [review]

**Review by UltraHDR — APPROVED** [review]

**Pull request merged**